storage/SystemInfo/CmdBtrfs.cc

```diff
@@ -109,11 +137,85 @@ namespace storage
     }
 
 
-    std::ostream& operator<<(std::ostream& s, const CmdBtrfsShow::Entry& entry)
+    std::ostream&
+    operator<<(std::ostream& s, const CmdBtrfsShow::Entry& entry)
     {
 	s << entry.devices;
 
 	return s;
     }
 
+
+    CmdBtrfsSubvolumes::CmdBtrfsSubvolumes(const string& mount_point, bool do_probe)
```

Comment:
The do_probe flag is not needed in the redesign since the SystemCmd support mocking.